01-Git_Bash_Make/Task2_bash_script/script

```diff
@@ -5,6 +5,7 @@ directory=/tmp/guesanumber
 if ! [ -d $directory ]
 then
 	cp -R /home/igor/gl-kernel-basecamp-2019/01-Git_Bash_Make/Task1_game $directory
+	tar -cvzf $directory/Task1_game.tar.gz $directory
```

Comment:
While not strictly prohibited, placing the tarball in the directory being archived is bad idea - the procedure loses reenterability.